utils/create-module.js

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env node
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+/**
+ * Add the module as a dependency in modules/_canvas-kit-<TYPE>/package.json.
+ */
+const addModuleAsDependencyByType = (cwd, moduleName, type) => {
+  const packagePath = path.join(cwd, `modules/_canvas-kit-${type}/package.json`);
+  const packageContents = fs.readFileSync(packagePath);
+  const json = JSON.parse(packageContents);
+
+  // Add module, sort dependencies, then write back out
+  json.dependencies[`@workday/canvas-kit-${type}-${moduleName}`] = '^3.0.0-alpha.2';
```

Comment:
Maybe there's a better way to pull the `'^3.0.0-alpha.2';` part from a versions constant somewhere?

Reply:
Agreed. The idea is to version lock packages, so you can pull from the root `package.json`. Is that correct @lychyi?

Reply:
Hey thanks for having a look @NicholasBoll :)

When you say root `package.json` are you referring to the one at the very top level? I don't see a version specified there or in the yarn.lock.

However, it occurs, the generator could just take a look at what's already in:
`modules/_canvas-kit-${type}/package.json`

so for React ones would be:
`modules/_canvas-kit-react/package.json`

and CSS would be:
`modules/_canvas-kit-css/package.json`

And just use whatever's there since I don't see how you'd be needing create a new module AND bump versions. Seems reasonable to me if you're good with it. If there was a `version: VERSION` constant somewhere it would be cleaner to use that, but, if not, I can do proposed solution.

Thoughts?

Reply:
I meant the module type package. For instance, `json.version` will be `3.0.0-alpha.2` here. The new module gets the same version as the exported module.

Reply:
When we add a new module, the export module will have to be bumped to include it (minor bump once out of alpha). When we bump a new version, all packages should be versioned the same. I think `json.version` is safe to use. There is no other source of truth.

Reply:
It seems the version should be a hard-coded `0.0.0`. That is treated special - it isn't actually pulled from npmjs.org and allows us to deploy through lerna when we're ready to publish